ERCS/erc-7390.md

```diff
@@ -0,0 +1,366 @@
+---
+eip: 7390
+title: Vanilla Options
+description: An Interface for Vanilla Options
```

Comment:
Your title and description need to include more information.Your title should at least hint at what is in this proposal, and your description needs to introduce more information than was available in the title alone.

Reply:
Improved in commit [47da247](https://github.com/ethereum/ERCs/pull/53/commits/47da24757feff3357df6b75b525604e0f04d05c0), do you think it's better now?